test/e2e/page-objects/pages/experimental-settings.ts

```diff
@@ -9,9 +9,12 @@ class ExperimentalSettings {
 
   private readonly experimentalPageTitle: object = {
     text: 'Experimental',
-    css: '.h4',
+    css: 'h4',
```

Comment:
just small nit, preferring tag over css if we are using a tag value, for better clarity? not sure if we have guidelines about this, but seems a general convention followed in most tests? What do you think?

```suggestion
    tag: 'h4',
```

Reply:
That makes sense, changed, thanks !